jbpm-services/jbpm-executor/src/test/java/org/jbpm/executor/BasicExecutorBaseTest.java

```diff
@@ -622,7 +622,11 @@ public void testPrioritizedJobsExecution() throws InterruptedException {
         assertNotNull(executedLow);
         assertEquals("low priority", executedLow.getKey());
         
-        assertTrue(executedLow.getTime().getTime() > executedHigh.getTime().getTime());
+        logger.info("executedLow: {}", executedLow.getTime().getTime());
+        logger.info("executedHigh: {}", executedHigh.getTime().getTime());
+        logger.info("exec difference: {}", (executedLow.getTime().getTime() - executedHigh.getTime().getTime()));
```

Comment:
Can you please delete this logging? We don't need this anymore since we verified our theory about the issue.